rules-xml/tests/src/test/resources/xsd-validation/xsd-url-multiple-schemas.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
this new file affects test https://github.com/windup/windup/blob/master/rules-xml/tests/src/test/java/org/jboss/windup/rules/xml/XMLFileNestedConditionTest.java#L127